utils/abcd_sampler.jl

```diff
@@ -27,9 +33,10 @@ c_max_iter = parse(Int, conf["c_max_iter"])
 coms = ABCDGraphGenerator.sample_communities(τ₂, c_min, c_max, n, c_max_iter)
 open(io -> foreach(d -> println(io, d), coms), conf["communitysizesfile"], "w")
 
-ξ = parse(Float64, conf["xi"])
 isCL = parse(Bool, conf["isCL"])
-p = ABCDGraphGenerator.ABCDParams(degs, coms, nothing, ξ, isCL, false)
+islocal = haskey(conf,"islocal") ? parse(Bool, conf["islocal"]) : false
+
+p = ABCDGraphGenerator.ABCDParams(degs, coms, μ, ξ, isCL, islocal)
```

Comment:
so here we allow passing `islocal=true` when ξ is passed - does the code throw an error later?

Reply:
Yes, it's handled in `src/graph_sampler.jl` - add additional error in above file to catch it sooner?

Reply:
if the error is informative then no need to add it. If the error is messy then please add

Reply:
Yes, it's straightforward 

> "when ξ is provided local model is not allowed"